### PR TITLE
LPS-134562 Add new workflow definition xml

### PR DESCRIPTION
**Pull request by seltong**

### CODE REVIEW

**Comment by seltong** [comment]
ci:test:relevant

File: modules/apps/portal-workflow/portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/internal/runtime/integration/test/WorkflowDefinitionManagerTest.java [hunk 1]
```diff
@@ -412,6 +412,18 @@ public void testValidateSingleApproverScriptedAssignmentDefinition()
 		_assertValid(inputStream);
 	}
 
+	@Test
+	public void testValidateTaskWithMoreThanOneTransitions() throws Exception {
+		InputStream inputStream = _getResourceInputStream(
```
Comment (rafaprax) [inline modules/apps/portal-workflow/portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/internal/runtime/integration/test/WorkflowDefinitionManagerTest.java:417]: Hey @seltong,

rename to `testValidateTaskWithMoreThanOneDefaultTransition`. please

**Review by rafaprax — CHANGES_REQUESTED** [review]
Hey @seltong,

Did you check the CE version? Probably you will need to add similar changes there. 

Please merge the last 2 commits since they are related to the same test.

Thx and nice job

**Comment by seltong** [comment]
ci:test:sf

**Comment by rafaprax** [comment]
Just started reviewing :)

**Review by rafaprax — APPROVED** [review]

**Comment by rafaprax** [comment]
resent here https://github.com/liferay-workflow/liferay-portal/pull/494

**Pull request closed without merging**